before.md

```diff
@@ -32,6 +35,7 @@
     - Conversely, you may want to avoid planning the sprint that clashes with another event
     if your team is going to be engaged elsewhere or have other commitments.
 - **Consider promoting the sprint.** Generally, you might want to promote your upcoming sprint to attract other contributors.
+    - Consider whether you need a system to track participation for the Sprint, e.g. a Google Form/Eventbrite page
```

Comment:
Similar to the comment above, I think an extra sentence here to describe how the organisers might decide whether tracking participation is necessary or not, would make this point complete. E.g. for in-person events it is often necessary to collect registrations to estimate numbers for booking a room/catering etc. But these are less relevant to virtual events. Under which circumstances do you think it would be helpful to track participation in advance?